sig-network/charter.md

```diff
@@ -0,0 +1,55 @@
+# SIG Network Charter
+
+This charter adheres to the conventions described in the [Kubernetes Charter README] and uses
+the Roles and Organization Management outlined in [sig-governance].
+
+## Scope
```

Comment:
This section isn't entirely clear about the distinction between what SIG Network itself provides (APIs, interfaces, kube-proxy, ...?), and what it enables other people to provide (CNI plugins, Ingress implementations, ...?)

Reply:
I adjusted the wording a bit, but I think the "In scope" and "Out of scope" sections below make this clear.